src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,41 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskList {
+    private List<Task> tasks;
+
+    public TaskList() {
+        this.tasks = new ArrayList<>();
+    }
+
+    public TaskList(List<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    public void setDone(int index) {
+        tasks.get(index).setCompleted();
+    }
+
+    public void remove(int index) {
+        tasks.remove(index);
+    }
+
+    public void add(Task task) {
+        tasks.add(task);
+    }
+
+    public Task get(int index) {
+        return tasks.get(index);
+    }
+
+    public int size() {
+        return tasks.size();
+    }
+
+    public List<Task> getTasks() {
+        return tasks;
+    }
+
+}
```

Comment:
This seems to function like an ArrayList with an extra setDone method. Would it be better to implement TaskList as an extension of ArrayList?